src/ape_init/_cli.py

```diff
@@ -9,8 +9,26 @@
 
 @click.command(short_help="Initalize an ape project")
 @ape_cli_context()
-@click.option("--github")
+@click.option(
+    "--github", metavar="GH_username/repo", help="Add github_username/repository to clone project"
+)
 def cli(cli_ctx, github):
+    """
+    Ape Init allows the user to create an ape project with
+    default folders and ape-config.yaml
+
+        project_name/\n
+            contracts/\n
+            tests/\n
+            scripts/\n
+            ape-config.yaml\n
+
+    ape-config.yaml is to manage project dependencies such as plugins.
+    Adding plugins is format sensitive.
```

Comment:
This may be superfluous. 

How about we just link to the project docs guide instead?

https://docs.apeworx.io/ape/stable/userguides/projects.html

Reply:
Agree, please just link to docs on config